src/NuGet.Core/NuGet.Common/PathUtil/PathUtility.cs

```diff
@@ -1,10 +1,48 @@
 ﻿using System;
+using System.Collections.Generic;
 using System.IO;
 
 namespace NuGet.Common
 {
     public static class PathUtility
     {
+        /// <summary>
+        /// Returns OrdinalIgnoreCase windows and mac. Ordinal for linux.
+        /// </summary>
+        /// <returns></returns>
+        public static StringComparer GetStringComparerBasedOnOS()
+        {
+            if (RuntimeEnvironmentHelper.IsWindows || RuntimeEnvironmentHelper.IsMacOSX)
```

Comment:
Sorry, in CodeFlow -- need to provide more context. Mac can be configured to be case sensitive.

---
In reply to: [66690541](https://github.com/NuGet/NuGet.Client/pull/673#discussion_r66690541) [](ancestors = 66690541)